.travis.yml

```diff
@@ -1,26 +1,47 @@
+sudo: false
+
 language: python
 
 python:
   - 2.7
   - 3.3
   - 3.4
+  - 3.5
 
 env:
-  - DJANGO="Django>=1.5,<1.6"
-  - DJANGO="Django>=1.6,<1.7"
-  - DJANGO="Django>=1.7,<1.8"
-  - DJANGO="Django>=1.8,<1.9"
+  - DJANGO=1.6
+  - DJANGO=1.7
+  - DJANGO=1.8
+
+matrix:
+  exclude:
+    - python: 3.5
+      env: DJANGO=1.6
+    - python: 3.5
+      env: DJANGO=1.7
+  include:
```

Comment:
I'm not sure why this `include` is needed. Isn't this ocmbo already covered by `python` * `env`?